### PR TITLE
refactor: use standard RFC4122 form of uuid with hyphens

### PR DESCRIPTION
**Pull request by sfwn**
#### What this PR does / why we need it:

see: https://en.wikipedia.org/wiki/Universally_unique_identifier#Format
refactor: uuid return form of xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx

#### Specified Reviewers:

/assign @Effet 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  UUID use standard RFC4122 format: xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx           |
| 🇨🇳 中文    |    UUID 使用 RFC4122 标准格式 ：xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx        |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkg/crypto/uuid/uuid.go [hunk 1]
```diff
@@ -17,16 +17,26 @@ package uuid
...
 
-// UUID 返回 uuid.
-func UUID() string {
-	return Generate()
+// LegacyUUID return legacy uuid.
+// Deprecated.
+func LegacyUUID() string {
+	u := uuid.NewV4()
+	return fmt.Sprintf("%x%x%x%x%x", u[:4], u[4:6], u[6:8], u[8:10], u[10:])
 }
```
Comment (Effet) [inline pkg/crypto/uuid/uuid.go:42]: I suggest not modify old func, and just add a new func name: `UUIDString`, then this pr will change smaller.

**Review by Effet — CHANGES_REQUESTED** [review]

File: pkg/crypto/uuid/uuid.go (hunk 1 above)
Comment (sfwn) [reply pkg/crypto/uuid/uuid.go:42]: `UUIDString` is ugly and unacceptable.

Comment (Effet) [reply pkg/crypto/uuid/uuid.go:42]: then `New` is good name.

Comment (sfwn) [reply pkg/crypto/uuid/uuid.go]: better than uuidstring.

**Review by Effet — APPROVED** [review]

**Pull request merged**